src/values/indexables/indexable-builder-versions.php

```diff
@@ -20,9 +20,9 @@ class Indexable_Builder_Versions {
 		'date-archive'      => self::DEFAULT_INDEXABLE_BUILDER_VERSION,
 		'general'           => self::DEFAULT_INDEXABLE_BUILDER_VERSION,
 		'home-page'         => 2,
-		'post'              => 2,
+		'post'              => 3,
 		'post-type-archive' => 2,
-		'term'              => 2,
+		'term'              => 3,
```

Comment:
I don't think we should up the version of terms, as per [this](https://yoast.slack.com/archives/C02NX2P7UKW/p1639041093007600?thread_ts=1639040870.006900&cid=C02NX2P7UKW).